fix weird link failures on centos7:
pdfium/core/fpdfdoc/cpdf_formcontrol.cpp:134: undefined reference to `fxcrt::StringViewTemplate<char>::StringViewTemplate(char const&)'
--- pdfium/core/fxcrt/string_view_template.h.orig	2025-11-11 13:26:05.268265314 +0100
+++ pdfium/core/fxcrt/string_view_template.h	2025-11-11 13:26:36.836918076 +0100
@@ -73,7 +73,7 @@
   // Deliberately implicit to avoid calling on every char literal.
   // |ch| must be an lvalue that outlives the StringViewTemplate.
   // NOLINTNEXTLINE(runtime/explicit)
-  constexpr StringViewTemplate(const CharType& ch) noexcept
+  StringViewTemplate(const CharType& ch) noexcept
       : m_Span(
             reinterpret_span<const UnsignedType>(pdfium::span_from_ref(ch))) {}
 
--- pdfium/core/fxge/cfx_face.cpp.orig	2025-11-11 13:42:28.181052508 +0100
+++ pdfium/core/fxge/cfx_face.cpp	2025-11-11 13:45:12.583343976 +0100
@@ -227,7 +227,7 @@
     case fxge::FontEncoding::kBig5:
       return FT_ENCODING_BIG5;
     case fxge::FontEncoding::kGB2312:
-      return FT_ENCODING_PRC;
+      return FT_ENCODING_GB2312;
     case fxge::FontEncoding::kJohab:
       return FT_ENCODING_JOHAB;
     case fxge::FontEncoding::kLatin1:
@@ -259,7 +259,7 @@
       return fxge::FontEncoding::kAppleRoman;
     case FT_ENCODING_BIG5:
       return fxge::FontEncoding::kBig5;
-    case FT_ENCODING_PRC:
+    case FT_ENCODING_GB2312:
       return fxge::FontEncoding::kGB2312;
     case FT_ENCODING_JOHAB:
       return fxge::FontEncoding::kJohab;
